README.md

```diff
@@ -1,2 +1,213 @@
 # bootstrap
+
 Foundation Bootstrap Team
+
+## About this document
+
+The Node.js and JS Foundations have announced an intent to merge into a single
+umbrella organization hosting the combined portfolio of projects currently
+under each separate organization.
+
+The Board's of each Foundation have asked the various projects to come together
+and provide specific feedback on:
+
+1. Whether the merger should proceed
+2. What the overall structure of a merged Foundation should look like
```

Comment:
if we label each item `1.` github markdown will automate numbering. Good way to future proof.